lib/http_clients/creditas/person_api.ex

```diff
@@ -0,0 +1,28 @@
+defmodule HttpClients.Creditas.PersonApi do
+  @moduledoc false
+
+  @spec client(String.t(), String.t()) :: Tesla.Client.t()
+  def client(base_url, bearer_token) do
+    headers = headers(bearer_token)
+
+    middlewares = [
+      {Tesla.Middleware.BaseUrl, base_url},
+      {Tesla.Middleware.Headers, headers},
+      Tesla.Middleware.JSON,
+      {Tesla.Middleware.Retry, delay: 1_000, max_retries: 3},
+      {Tesla.Middleware.Timeout, timeout: 120_000},
+      Tesla.Middleware.Logger,
+      Goodies.Tesla.Middleware.RequestIdForwarder
```

Comment:
```suggestion
      Tesla.Middleware.Logger
```
acho que não tem esse header na doc da Creditas